.circleci/config.yml

```diff
@@ -179,7 +179,7 @@ jobs:
       - run:
           name: Push versioned docker containers
           command: |
-            pip3 install $(cat requirements.txt | grep -e awscli)
+            pip3 install $(cat conda.txt | grep -e awscli)
```

Comment:
shouldn't this be `pip.txt`?

Reply:
awscli package specification was originally in requirements.txt, then moved to conda.txt in this commit daa16a8a7372ba1ec27d561a80470a5e87e4c3cd, so get it from conda.txt now.

One thing to notice is: this step happens inside CI container, not the craftassist docker container which we built inside CI container (`pip.txt` is created in by copying requirements.txt from CI container into craftassist docker container)

Reply:
okay. in that case, the correct fix is to move the `awscli` entry back into `requirements.txt`.

`conda.txt` only contains conda package specifications. It's "luck" that `awscli` has the same specification for both `pip` and `conda`.